src/sentry/static/sentry/app/views/organizationContext.jsx

```diff
@@ -176,7 +176,7 @@ const OrganizationContext = createReactClass({
       this.props.api,
       this.getOrganizationSlug(),
       this.props.detailed,
-      true // silent, to not reset a lightweight org that was fetched
+      isSilent // silent, if true, will not reset a lightweight org that was fetched
```

Comment:
Can we make sure that we don't reset a lightweight org e.g. when we go from a route that is in the lightweight route tree vs normal route tree.

Reply:
I attached a debugger to this. Seems like this code path is taken when
1. Loading the app for the first time
2. Org switch

Switching between the trees doesn't seem to trigger this code path